.ci/scripts/release/maven-release.sh

```diff
@@ -11,5 +11,6 @@ mvn -s ${MAVEN_SETTINGS_XML} release:prepare release:perform -B \
     -DpushChanges=${PUSH_CHANGES} \
     -DremoteTagging=${PUSH_CHANGES} \
     -DlocalCheckout=${SKIP_DEPLOY} \
+    -DcompletionGoals="spotless:apply" \
     -P!autoFormat \
-    -Darguments='--settings=${MAVEN_SETTINGS_XML} -P-autoFormat -DskipChecks=true -DskipTests=true -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'
+    -Darguments='--settings=${MAVEN_SETTINGS_XML} -P-autoFormat -DskipTests=true -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'
```

Comment:
❌ Why did you re-enable the checks? That would slow down the release process. We also skip the tests. The assumption is that the release is checked and tested in the steps already before the maven release is published.

Reply:
> The comments https://github.com/camunda/zeebe/issues/10602#issuecomment-1329376912 mention the deletion of the -DskipChecks=true

Tl;dr, `skipChecks` means spotless gets skipped.

Reply:
🤔 Let me try something else. Maybe I can change the goal to `spotless:apply -DskipChecks=false' to still skip it during the process, but not skipping spotless.

Reply:
Unfortunately that doesn't work, so I think it's worth removing the `skipChecks`. The time saved is pretty minor either way.

Reply:
Aha, that makes sense. I missed this line from the PR description.

🔧 It would be helpful to split it off into a separate commit explaining why it's necessary.

🔧 to keep the speed the same, we could also explicitly skip each of the unwanted checks:
```
-Dcheckstyle.skip=true
-Drevapi.skip=true
-Dlicense.skip=true
-Denforcer.skip=true
-Dmdep.analyze.skip=true
```

🔧 Another idea is to disable the skipping of the wanted plugins explicitly:
```
-DskipChecks=true
-Dsort.skip=false
-Dspotless.apply.skip=false
-Dspotless.checks.skip=false
```

Reply:
> to keep the speed the same, we could also explicitly skip each of the unwanted checks instead of not skipping any checks:

We use the `-DskipChecks` in so many places. I really don't want to modify them all to skip all plugins independently. It's also annoying if we add a new check because we'd need to update commands everywhere

> Another idea is to disable the skipping of the wanted plugins explicitly, but I'm not sure whether this will work because these values are defined in the parent/pom.xml through skipChecks:

I'll give it a try with adding '-Dspotless.apply.skip=false'. I could see why that would work.

Reply:
💭 I just noticed that the `sort.skip` is no longer used 💡

Reply:
`-Dspotless.apply.skip=false` seems to work.

The build step is ~2min faster than without: 
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/5787702/204537877-cc7151e1-6fb2-47db-bcf6-b71fc7d28ebf.png">

The formatting still gets applied before the commit:
https://github.com/camunda/zeebe/commit/109d18f4ea4d6f4ac655067caaba7fce05458110

Reply:
I've also added some explanation surrounding it to the commit message.